docs/rfcs/UserProfilesv2_schema.json

```diff
@@ -218,130 +286,44 @@
                 "metadata",
                 "value"
```

Comment:
And this
```
                 "vaue": { "$ref": "#/definitions/ValueElement"}
```

Reply:
no, refs are for "object types", this is for which attribute is required